daemon/network/settings.go

```diff
@@ -43,4 +43,5 @@ type Settings struct {
 	SandboxKey             string
 	SecondaryIPAddresses   []Address
 	SecondaryIPv6Addresses []Address
+	AnonymousEndpoint      bool
```

Comment:
Just a nit while reading, and don't know if we actually have a convention for this, but I like names such as `IsXXX / `HasXXX` for boolean values (in this case e.g. `IsAnonymous`)